esmvaltool/diag_scripts/ipcc_ar5/ch12_calc_IAV_for_stippandhatch.ncl

```diff
@@ -0,0 +1,426 @@
+; #############################################################################
+; INTERANNUAL VARIABILITY MULTI-MODEL MEAN FOR STIPPLING SIGNIFICANCE IPCCCH12
+; Author: Ruth Lorenz (ETH Zurich, Switzerland)
+; CRESCENDO project
+; #############################################################################
+;
+;  Description
+;     Calculate Interannual variability from piControl runs for plotting
+;     significance with stippling and hatching, save for each model
+;     (iavmode: "each") or multi-model mean only.
+;     Can either be calculated over full time period of piControl run
+;     (no periodlength given)
+;     or first averaged annually/seasonally over periodlength and then standard
+;     deviation calculated based on averaged periods.
+;
+;  Required diag_script_info attributes (diagnostics specific)
+;     time_avg:  time period to calculate IAV, e.g. annualclim, seasonalclim
+;
+;  Optional diag_script_info attributes (diagnostic specific)
+;     iavmode: calculate multi-model mean of IAV over all models or calculate
+;              and save IAV for each individual model? (mmm, each)
+;     periodlength: length of periods to calculate IAV accross, depends on
+;                   period lengths used in
+;                   ch12_calc_map_diff_mmm_stippandhatch.ncl
+;                   if not given whole time period calculated at once
+;
+;  Caveats
+;     Needs lots of memory for 3D ocean variables
+;
+;  Modification history
+;     20161219-A_lore_ru: remove seasonal cycle before std if seasonal
+;     20161024-A_lore_ru: adapted to ESMValTool
+;     20130501-A_sedl_ja: written for IPCC AR5 as get_natvar.ncl.
```

Comment:
This author is not listed in `config-references.yml`.

Reply:
True, Jan wrote all the original versions of the scripts for IPCC AR5, so he does deserve some credit. But I did not add him in the config_references given that he hasn't done anything with the ESMValTool and is not at ETH anymore. Suggestions?

Reply:
Just remove this line and modify the second line as "adapted to ESMValTool based on original scripts by ...".